libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/editor/DBClientInfoEditor.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.tools.dbbrowser.editor;
+
+import com.oceanbase.tools.dbbrowser.model.DbClientInfo;
+
+public interface DBClientInfoEditor {
+
+    boolean setClientInfo(DbClientInfo clientInfo);
```

Comment:
it seems make more sense if add setClientInfo into SessionExtensionPoint

Reply:
I also think so,  but the current plugin module only supports loading by dialectType,  whereas here version is also necessary.

Reply:
we can also add `clientInfoSupported` method in extension point

Reply:
good idea!